src/components/structures/LoggedInView.tsx

```diff
@@ -94,7 +94,7 @@ interface IProps {
 
 interface IUsageLimit {
     // eslint-disable-next-line camelcase
-    limit_type: "monthly_active_user" | string;
+    limit_type: "monthly_active_user" | "hs_disabled" | string;
```

Comment:
Where is this specced? Given that its not a namespaced value it should be somewhere :(

https://matrix.org/docs/spec/client_server/r0.6.1#id165

Reply:
Good question. it really shouldn't have ended up in synapse without a namespace.

Reply:
Coming back to this, `limit_type`is not specced but used in synapse as per https://github.com/matrix-org/synapse/issues/9237.

I'd like to merge this to fix usability from a pragmatism perspective, on the basis that this is just an extra mapping of a string enum to some human readable text and is unlikely to result in unusual behaviour

Reply:
Can you stick a comment above this line with bit of text + issue link please

Reply:
Added